### PR TITLE
Added a !plugin info admin command.

### PR DESCRIPTION
**Pull request by gbin**
The command allows you to get the all the states, namespaces, log etc...
from a plugin.

### CODE REVIEW

**Comment by sijis** [comment]
You seem to be missing a docstring :)
```
• .plugin info - (undocumented)
```

It seems to fail if the plugin is disabled:
```
[@saviles ➡ @zombie] >>> .plugin info VersionChecker
Computer says nooo. See logs for details:
'NoneType' object has no attribute 'keys'
```

I like this very much!!

A couple of things that may need adjustment
1.  `log destination: errbot.plugins.Health` - I'm not sure what `log destination` means in this context. I'm guessing is the logging module reference. At first glance i was expecting a log file. So in short, this is a little confusing
2. I wonder if having the `.plug` path as a separate item (like `full_module_path`) makes thing more uniform.
```
Plugin info
 name: Health
 module: health
 full_plug_path: /home/saviles/data/git/github/errbot/errbot/core_plugins/health.plug
 full_module_path: /home/saviles/data/git/github/errbot/errbot/core_plugins/health.py
...

File: errbot/core_plugins/plugins.py [hunk 1]
```diff
@@ -303,3 +304,14 @@ def plugin_unblacklist(self, _, args):
                 return f'Error activating plugin: {pae}'
 
         return self._bot.plugin_manager.unblacklist_plugin(args)
+
+    @botcmd(admin_only=True, template='plugin_info')
+    def plugin_info(self, _, args):
+        pm = self._bot.plugin_manager
```
Comment (sijis) [inline errbot/core_plugins/plugins.py]: Need docstring.

**Comment by sijis** [comment]
Ohh, also noticed that `storage content` is always showing

```
 storage content
┏━━━━━┳━━━━━━━┓
┃ Key ┃ Value ┃
┡━━━━━╇━━━━━━━┩
│     │       │
└─────┴───────┘
```

**Comment by gbin** [comment]
ha because it is empty, this is cosmetic
once your plugin wrote something in its storage you see something there
[@gbin ➡ @errbot] >>> !plugin info Poll
Plugin info from /home/gbin/projects/err-plugins/err-poll/poll.plug
 name: Poll
 module: poll
 full_module_path: /home/gbin/projects/err-plugins/err-poll/poll.py
 core: False
 class: errbot.plugins.poll.Poll
 storage namespace: Poll
 log destination: errbot.plugins.Poll
 log level: NOTSET
 storage content
┏━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃ Key          ┃ Value                                                                                           ┃
┡━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│ current_poll │ hey                                                                                             │
├──────────────┼─────────────────────────────────────────────────────────────────────────────────────────────────┤
│ polls        │ {&#39;hey&#39;: &lt;errbot.plugins.poll.PollEntry object at 0x7f24a6df0e10&gt;} │
└──────────────┴─────────────────────────────────────────────────────────────────────────────────────────────────┘

**Pull request merged**